### DISCUSSION TITLE
Welcome to DurableFunctionsMonitor Discussions!

### DISCUSSION THREAD

**Discussion by scale-tone** [microsoft/DurableFunctionsMonitor#85] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hysnyasir** [comment]
Issues with VS Code Durable Function Monitor Extension (Value cannot be null. (Parameter 'provider')"

I am having an issue while using the Durable Function Monitor extension. When I try to open any durable function it gives me the below error "Func: Value cannot be null. (Parameter 'provider')"

Solution is implemented in Visual Studio 2022. I am using VS Code to use the Durable Function Monitor Extensions. I also installed the Azure tool extension.

**Reply by scale-tone** [reply]
@hysnyasir , please, open `Durable Functions Monitor` output channel:

![image](https://github.com/microsoft/DurableFunctionsMonitor/assets/5447190/319afe36-596c-47c1-b6fc-2e517e72085a)

and post here what you see in there (excluding any secrets).

**Comment by hysnyasir** [comment]
@scale-tone Please find below error detail:


Warning: Proxies are not supported in Azure Functions v4. Instead of 'proxies.json', try Azure API Management: https://aka.ms/AAfiueq
[2023-06-20T17:04:24.579Z] Csproj not found in c:\Users\extcikyam\.vscode\extensions\durablefunctionsmonitor.durablefunctionsmonitor-6.2.0\backend directory tree. Skipping user secrets file configuration.
[2023-06-20T17:04:25.675Z] A host error has occurred during startup operation 'a2e85996-7dff-44a3-934d-e8f6c52f932e'.
[2023-06-20T17:04:25.676Z] Microsoft.Azure.WebJobs.Extensions.Http: Could not load file or assembly 'System.Net.Http.Formatting, Version=5.2.8.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.
ERROR: Value cannot be null. (Parameter 'provider')

**Comment by scale-tone** [comment]
@hysnyasir ,  the error indicates the issue previously covered by these two bugs:
https://github.com/microsoft/DurableFunctionsMonitor/issues/76
https://github.com/microsoft/DurableFunctionsMonitor/issues/75

See the instructions there on how to resolve it (basically you need to install the latest version of Azure Functions Core Tools and make sure it takes effect).

But also the latest DfMon should automatically check the Core Tools version and show a different, more specific error message, if it is stale. To understand why it is not happening in your case, I need to see **the entire** log, not just the last few lines.

**Comment by hysnyasir** [comment]
@scale-tone 
I have re-installed the latest Azure Function Core tools and I also installed lasted .Net 6 Sdk. Now, the Durable Function Monitor is working for me.

Thanks for your help.
